tests/clv/models/test_beta_geo.py

```diff
@@ -324,3 +325,16 @@ def test_model_repr(self):
             "\nr~HalfFlat()"
             "\nlikelihood~Potential(f(r,alpha,b,a))"
         )
+
+    @pytest.mark.parametrize("fit_method", ["mcmc", "map"])
+    def test_sample_population_distributions(self, fit_method: str):
+        model = BetaGeoModel(
+            customer_id=self.customer_id,
+            frequency=self.frequency,
+            recency=self.recency,
+            T=self.T,
+        )
+
+        model.fit(fit_method=fit_method)
+        samples = model.sample_population_distributions()
+        assert isinstance(samples, xr.Dataset)
```

Comment:
Can we assert something more concrete, like the mean/std is close enough to what would be expected from the fit?

Also let's not do an actual fit which would be a relatively slow test. You can borrow the logic in the other tests of doing a fake fit from prior_predictive

Reply:
Yes. Good idea